platform/windows/display_server_windows.cpp

```diff
@@ -3382,11 +3427,17 @@ DisplayServer::WindowID DisplayServerWindows::_create_window(WindowMode p_mode,
 				WindowRect.top,
 				WindowRect.right - WindowRect.left,
 				WindowRect.bottom - WindowRect.top,
-				nullptr, nullptr, hInstance, nullptr);
+				nullptr,
+				nullptr,
+				hInstance,
```

Comment:
Not directly related to your changes but while at it, I'd suggest removing the commented out code above which should obviously not be uncommented (would add two extra parameters to the function call).

Reply:
I figured reduzio left it there for a reason, maybe to consider using it again later?  I don't have sufficient ownership to remove their stuff.